sportnet/sportnet/playgrounds/models.py

```diff
@@ -4,14 +4,24 @@
 
 
 class PlayGround(TimeStampedModel):
+
+    title_action = models.CharField(max_length=128, verbose_name='Название мероприятия', blank=False, null=False)
+    admins = models.ManyToManyField(get_user_model(), verbose_name='Администратор',
```

Comment:
get_user_model() крайне не рекомендуется использовать в моделях.

> When you define a foreign key or many-to-many relations to the user model, you should specify the custom model using the AUTH_USER_MODEL setting. For example:

```suggestion
    admins = models.ManyToManyField(settings.AUTH_USER_MODEL, verbose_name='Администратор',
```
Не забудь `from django.conf import settings`